app/services/encryption/password_verifier.rb

```diff
@@ -73,7 +73,7 @@ def create_digest_pair(password:, user_uuid:)
     def verify(password:, digest_pair:, user_uuid:)
       digest = digest_pair.multi_or_single_region_ciphertext
       password_digest = PasswordDigest.parse_from_string(digest)
-      return verify_uak_digest(password, digest) if stale_digest?(digest)
+      return verify_uak_digest(password, digest) if password_digest.uak_password_digest?
```

Comment:
This is not strictly related to the logging, but I think it makes it easier to understand and skips an extra parse